cloud-sql/mysql/servlet/src/main/java/com/example/cloudsql/InputValidator.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.cloudsql;
+
+import java.util.Locale;
+import javax.annotation.Nullable;
+
+public class InputValidator {
```

Comment:
nit: You could merge `InputValidator` and `DatabaseSetup` into a utils class instead of each having it's own file if you wanted